app/controllers/tags_controller.rb

```diff
@@ -80,7 +80,7 @@ def suggest
   private
 
   def tags
-    @tags ||= Tag.direct.includes(:sponsorship).limit(100)
```

Comment:
We were showing tags that had sponsorship, but I've reverted the change and used the previous business logic which is to order the tags by their hotness score.